src/cargo/core/resolver/mod.rs

```diff
@@ -859,8 +891,9 @@ fn generalize_conflicting(
             // A dep is equivalent to one of the things it can resolve to.
             // Thus, if all the things it can resolve to have already ben determined
             // to be conflicting, then we can just say that we conflict with the parent.
+            let first_minimal_version = false;
```

Comment:
Why is it okay to pass false here? We only get to this code path if this package has a parent, and therefore we know that `first_minimal_version` should be false.

But a comment would probably be helpful here.